### PR TITLE
Feature/review-later

### PR DESCRIPTION
**Pull request by PinkhamSoftware**
What:
- Create JWT Token for meeting
- Read and validate JWT token for meeting
- Get Meeting details
- SignOff meeting

Why:
- So meetings can be reviewed later

Notes:
- Issues copying appsettings.json that why tests are not building

### CODE REVIEW

File: ManageATenancyAPI.Tests/v2/AcceptanceTests/SaveETRAMeeting/SaveEtraMeetingAcceptanceTests.cs [hunk 1]
```diff
@@ -109,7 +97,7 @@ public async Task can_save_etra_meeting_with_issues(int attendees, string issueT
             //act
             var response = await _classUnderTest.Post(inputModel).ConfigureAwait(false);
             //assert
-            var outputModel = response.GetOKResponseType<SaveETRAMeetingOutputModel>();
+            var outputModel = response.GetOKResponseType<SaveEtraMeetingOutputModelOutputModel>();
```
Comment (danielwilliams94) [inline ManageATenancyAPI.Tests/v2/AcceptanceTests/SaveETRAMeeting/SaveEtraMeetingAcceptanceTests.cs]: Assuming "SaveEtraMeetingOutputModelOutputModel" is deliberate?

Comment (PinkhamSoftware) [reply ManageATenancyAPI.Tests/v2/AcceptanceTests/SaveETRAMeeting/SaveEtraMeetingAcceptanceTests.cs]: nope, that's a resharper thing

Comment (danielwilliams94) [reply ManageATenancyAPI.Tests/v2/AcceptanceTests/SaveETRAMeeting/SaveEtraMeetingAcceptanceTests.cs]: Ah good, I'm glad, seemed a bit funky 😄

Comment (PinkhamSoftware) [reply ManageATenancyAPI.Tests/v2/AcceptanceTests/SaveETRAMeeting/SaveEtraMeetingAcceptanceTests.cs]: fixed

**Pull request merged**